data/database.py

```diff
@@ -755,6 +756,26 @@ class RobotAccountToken(BaseModel):
     fully_migrated = BooleanField(default=False)
 
 
+class QuotaLimitGroups(BaseModel):
+    group_name = CharField()
```

Comment:
I'm still not sure what "LimitGroups" are needed for.

Reply:
They were an abstraction I thought I needed to start with but as development went on I realized they were unnecssary they have been refactored out of the code.